rest-service/manager_rest/rest/resources_v3_1/deployments.py

```diff
@@ -1260,7 +1257,7 @@ def _make_new_group_deployment(self, rm, group, new_dep_spec, count,
         new_id, is_id_unique = self._new_deployment_id(group, new_dep_spec)
         inputs = new_dep_spec.get('inputs', {})
         labels = rest_utils.get_labels_list(new_dep_spec.get('labels') or [])
-        labels += group_labels
+        labels.extend([Label(*label) for label in group_labels])
```

Comment:
also btw, if you use `.extend` rather than `+=` for lists, then no need to make it a listcomp that just wastes memory. Just `.extend(Label() for label in group_labels)` is fine

Reply:
Updated.